conda_vendor/conda_vendor.py

```diff
@@ -574,14 +591,34 @@ def _get_system_virtual_packages(solver: str) -> List[dict]:
     conda_info = check_output(split(f"{solver} info --json"), text=True)
     _json = json.loads(conda_info)
     virtual_packages = []
-    for pkg in _json.get("virtual_pkgs", []):
-        virtual_packages.append(
-            {"name": pkg[0], "version": pkg[1], "build_string": pkg[2]}
-        )
 
+    if not is_micromamba(solver):
+        for pkg in _json.get("virtual_pkgs", []):
+            virtual_packages.append(
+                    {"name": pkg[0], "version": pkg[1], "build_string": pkg[2]}
+                    )
+    else:
```

Comment:
Is this what was breaking on Isaacs machine?

Reply:
no.  on isaac's machine he hadn't correctly setup his zsh, so wasn't getting his PATH set correctly.